test/github-issues/296/issue-296.ts

```diff
@@ -0,0 +1,67 @@
+import "reflect-metadata";
+import {closeTestingConnections, createTestingConnections, reloadTestingDatabases} from "../../utils/test-utils";
+import {Connection} from "../../../src";
+import {Post} from "./entity/Post";
+import {expect} from "chai";
+import {User} from "./entity/User";
+import {Comment} from "./entity/Comment";
+
+
+describe("github issues > #296 select additional computed columns", () => {
+
+    let connections: Connection[];
+    before(async () => connections = await createTestingConnections({
+        entities: [__dirname + "/entity/*{.js,.ts}"],
+        enabledDrivers: ["mysql"] // we can properly test lazy-relations only on one platform
```

Comment:
Why is this the case? We should test this across all drivers.

Reply:
I only make changes in MySQL Driver, for this I enable this driver alone. If you prefer I can enable all others drivers.

Reply:
yes, we need to make sure this is supported across all drivers.

Reply:
If I've only touched the MYSQL driver, it shouldn't affect the other drivers. I only enabled MYSQL because I can't test the other environments in local.

Reply:
You touched the base driver.  You updated all drivers